packages/api/ecs/async-operation/tests/test-index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const test = require('ava');
-const uuidv4 = require('uuid/v4');
+const { v4: uuidv4 } = require('uuid');
```

Comment:
Was previously getting error
```
[Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './v4' is not defined by "exports"](https://stackoverflow.com/questions/62549457/error-err-package-path-not-exported-package-subpath-v4-is-not-defined-by)
``` 
so I changed the import syntax

Reply:
Weird that that would have caused an issue here but that syntax is totally fine so 🤷